radiography/src/main/java/radiography/ViewStateRenderers.kt

```diff
@@ -94,4 +94,17 @@ public object ViewStateRenderers {
       }
     }
   }
+
+  /**
+   * Creates a [ViewStateRenderer] that renders views of type [T].
+   */
+  @JvmStatic
+  @PublishedApi internal fun <T : Any> viewStateRendererFor(
```

Comment:
not public?

Reply:
Nope, see reply to your comment above.

Reply:
If we actually need to call this one from Kotlin at some point, we can always change the visibility to public in a minor release.